x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/CcrIntegTestCase.java

```diff
@@ -311,7 +311,8 @@ public Path nodeConfigPath(int nodeOrdinal) {
     @Override
     public List<String> filteredWarnings() {
         return Stream.concat(super.filteredWarnings().stream(),
-            List.of("Configuring multiple [path.data] paths is deprecated. Use RAID or other system level features for utilizing " +
```

Comment:
IIRC path.shared_data is randomly set by the test framework. I think in that case this should be in the ESTestCase filtered warnings?

Reply:
I wasn't aware that it was randomly set. I will move it to ESTestCase